### ISSUE TITLE
Ajour will not recognize Addon downgrades

### ISSUE THREAD

**Issue by Silun** [ajour/ajour#722]
**Describe the bug**
After updating ElvUI from 2.10 to 2.11 and manually downgrading to 2.10 from a backup, Ajour still detects it as 2.11 and won't update again.

**To Reproduce**
1. Update ElvUI from 2.10 to 2.11 via Ajour
2. See how that update fucks ElvUI up, restore ElvUI + options addon from last backup
3. Restart Ajour

**Expected behavior**
Ajour should find the version to be 2.10 as reported ingame

**Screenshots**
![grafik](https://user-images.githubusercontent.com/17919060/129117893-2ace7e08-e351-4802-8a1e-9aecf1b20680.png)

**Software involved**
Please complete the following information:
 - OS: Win10
 - Ajour version: 1.3.0
 - (if applicable) ElvUI 2.11 for TBCC

**Additional context**

**Log Output**
[ajour.log](https://github.com/ajour/ajour/files/6971956/ajour.log)

**Comment by casperstorm** [comment]
Did you delete it through Ajour?

**Comment by Silun** [comment]
No, because I have it set to delete the saved variables on addon deletion, and since I only wanted to change the addon itself but leave the variables untouched, I manually removed the 2.11 addon folders and then copied the 2.10 folders from the backup.

**Comment by d3m4** [comment]
> 
> 
> No, because I have it set to delete the saved variables on addon deletion, and since I only wanted to change the addon itself but leave the variables untouched, I manually removed the 2.11 addon folders and then copied the 2.10 folders from the backup.

Had the same issue, for some reason while switching from windows to mac it corrupted everything; got an older backup going and now have this save problem;

If capturing the right version ins an issue, an option to "force download" would "fix the problem" as well;

Love Ajour; cheers!
